include/boost/exception/detail/error_info_impl.hpp

```diff
@@ -46,6 +47,13 @@ boost
         typedef T value_type;
 
         error_info( value_type const & value );
+#ifndef BOOST_NO_CXX11_RVALUE_REFERENCES
+        error_info( value_type && value);
+        error_info( error_info const & ) = default;
```

Comment:
Check for `BOOST_NO_CXX11_DEFAULTED_FUNCTIONS` and define function bodies by hand in worst case.